terasoluna-gfw-parent/pom.xml

```diff
@@ -489,7 +489,7 @@
     <org.mybatis.version>3.5.11</org.mybatis.version>
     <org.mybatis.spring.version>3.0.1</org.mybatis.spring.version>
     <!-- == MapStrcut== -->
-    <mapstruct.version>1.5.3.Final</mapstruct.version>
+    <mapstruct.version>1.5.2.Final</mapstruct.version>
```

Comment:
- [ ] mapstruct should be 1.5.3

Reply:
Fixed in the following commit.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1217/commits/bc71e46813aee1b430b24849e1a24cdea8985423